src/styles/blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+
+@import "./stars.scss";
+
+.cards {
+  display: flex;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  max-width: 944px;
+  flex-shrink: 0;
+  margin: auto;
```

Comment:
Follow the checklist, please 
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/99403198/229285034-4bd2ba4e-0908-401f-9e01-9ee0bf03ec2e.png">